packages/@okta/vuepress-site/docs/concepts/direct-authentication/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Direct authentication
+meta:
+  - name: description
+    content: An overview of what direct authentication is and how you can use it.
+---
+
+# Direct authentication
+
+<!-- Direct authentication simplifies the authentication flow, enhances user experience by minimizing redirects, and provides developers with greater control over security measures. -->
+## What is Direct authentication
+
+[Direct authentication](/docs/guides/configure-direct-auth-grants/aotp/main/) is an authentication method that allows users to directly exchange their credentials (such as usernames and passwords) with an app or service without the need for intermediary identity providers or complex redirection flows.
+
+In direct authentication, the client app communicates directly with the authentication service to validate user credentials. This method is useful in scenarios where usability constraints hinder the use of browser-based flows, such as with mobile apps. In addition, it's beneficial in situations where speed and simplicity are important, such as with single-page applications (SPAs). By eliminating the need for redirects to external identity providers, direct authentication reduces latency and enhances the overall user experience.
+
+### Provides more control
+
+Direct auth gives you more control over the authentication process by handling authentication internally in your app. You can tailor security measures to your needs without relying on third-party providers. Keep in mind that this also places a greater responsibility on you to ensure the secure handling of user credentials. Use direct authentication only in situations where there's a high degree of trust between the user and your app.
+
+## Key Characteristics
+
+Direct authentication stands apart from traditional methods due to several defining characteristics that streamline the user experience and simplify implementation in specific scenarios:
+
+* **Client-initiated credential exchange**: The client app directly sends user credentials (for example, username/password, API key, token) to the service's authentication endpoint.
+* **API-centric**: Authentication is typically performed using RESTful APIs.
+* **Minimal redirection**: The need for browser redirects to IdPs is reduced or eliminated. This keeps the user within the app's context.
+* **Direct response**: The authentication service directly responds to the client with an authentication token (for example, JWT or OAuth token) upon successful validation.
```

Comment:
I think this line can be removed. Other apis also just return tokens as well. If you do decide to keep it, the only thing returned is OAuth/OIDC tokens